webpack.config.js

```diff
@@ -31,6 +37,11 @@ module.exports = {
         'NODE_ENV': `"${process.env.NODE_ENV}"`
       }
     }),
-    new CommonsChunkPlugin("common.js", ["popup", "content_script"])
+    new CommonsChunkPlugin("common.js", ["popup", "content_script"]),
+    new CopyWebpackPlugin([
+      { from: 'manifest.json' },
+      { from: 'index.html' },
+      { from: 'styles.css' }
```

Comment:
I thought about it, but I generally saw dist being not gitignored, so I didn't remove it. Will make it so.